src/sections/pages/users/usersTabs.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+    Column, Tab, Table, Tabset,
+} from 'react-rainbow-components';
+import UserCell from '../../../components/UserCell';
+import Status from './status';
+import TablePagination from '../../../components/TablePagination';
+
+export default class UsersTabs extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            activeTabName: 'allUsers',
+            activePage: 1,
+        };
+        this.handleOnSelect = this.handleOnSelect.bind(this);
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    getTableData() {
+        const { users } = this.props;
+        const activeUsers = users.filter(user => user.status === 'ONLINE');
+        const { activeTabName, activePage } = this.state;
+        const firstItem = (activePage - 1) * 8;
+        const lastItem = firstItem + 8;
+        if (activeTabName === 'allUsers') {
+            return users.slice(firstItem, lastItem);
+        }
+        return activeUsers.slice(firstItem, lastItem);
+    }
+
+    getPages() {
+        const { users } = this.props;
+        const activeUsers = users.filter(user => user.status === 'ONLINE');
```

Comment:
this can be a function since it is called twice so:
create a function getActiveUsers, put it in another file and test it